### PR TITLE
Implemented missing r_debug_native_kill for win32

### PR DESCRIPTION
**Pull request by yetmorecode**
Implementation for native win32 to terminate a process was completely missing.

### CODE REVIEW

File: libr/debug/p/debug_native.c [hunk 1]
```diff
@@ -1169,23 +1169,11 @@ static int r_debug_native_bp_read(int pid, ut64 addr, int hw, int rwx) {
 
 // TODO: implement own-defined signals
 static int r_debug_native_kill(RDebug *dbg, int pid, int tid, int sig) {
+	int ret = R_FALSE;
+	if (pid==0) pid = dbg->pid;
```
Comment (ret2libc) [inline libr/debug/p/debug_native.c]: ```pid == 0```

File: libr/debug/p/native/w32.c [hunk 2]
```diff
@@ -615,4 +615,29 @@ RList *w32_pids (int pid, RList *list) {
...
+	/* stop debugging if we are still attached */
+	DebugActiveProcessStop (pid);
+	if (TerminateProcess (process, 1) == 0) {
+		print_lasterr ((char *)__FUNCTION__, "TerminateProcess");
+		return R_FALSE;
+
+	}
+	DWORD ret_wait;
+	/* wait up to one second to give the process some time to exit */
+	if ((ret_wait = WaitForSingleObject (process, 1000)) != 0) {
```
Comment (ret2libc) [inline libr/debug/p/native/w32.c]: Does 0 mean WAIT_OBJECT_0 ? If so, it would be better to use that i think

File: libr/debug/p/native/w32.c [hunk 3]
```diff
@@ -615,4 +615,29 @@ RList *w32_pids (int pid, RList *list) {
...
+		if (ret_wait == WAIT_TIMEOUT) {
+			eprintf ("(%d) Waiting for process to terminate timed out.\n",
+						pid);
```
Comment (ret2libc) [inline libr/debug/p/native/w32.c]: remove two tabs. We use only one tab to indent multilines.

**Comment by XVilka** [comment]
@yetmorecode just a question - you are testing this code on cmd.exe + mingw32? Or some another way?

**Comment by XVilka** [comment]
Please also refer to the https://github.com/radare/radare2/issues/1194 - it's a tracker of the Windows support.

**Comment by yetmorecode** [comment]
@XVilka I cross compile from a linux VM with x86_64-w64-mingw32-gcc. Then use radare.exe with cmd.exe or the r_*.dlls

File: libr/debug/p/native/w32.c (hunk 2 above)
Comment (yetmorecode) [inline libr/debug/p/native/w32.c]: Good catch. There is actually no "success" return return value like 0. I will change it.

**Comment by radare** [comment]
merged!

**Pull request closed without merging**